codemod/output/AwsSignatureV4.ts

```diff
@@ -0,0 +1,199 @@
+import { AwsSignature } from "./AwsSignature";
+import { uri, awsUrl } from "./Utils";
+import { Global } from "./Global";
+
+class AwsSignatureV4 extends AwsSignature {
+	public _cr: any;
+	public payload: any;
+
+  constructor(request) {
+    super(request);
+    this._cr = undefined;
```

Comment:
`_cr` can't be defined in his declaration?

Reply:
There's that kind of stuff in other files.

Reply:
Done, I think that I removed or moved all the initializations to their definition in the class.